openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -21,30 +21,29 @@ <h2 class="step__title">
                         {% endif %}
                     </span>
                 </button>
-            </h2>
+            </h4>
         </header>
 
         <div class="ui-slidable__content" id="oa_grade_{{ xblock_id }}_content" aria-labelledby="oa_grade_{{ xblock_id }}">
             <div class="wrapper--step__content">
                 <div class="step__content">
                     <article class="submission__answer__display step__content__section">
-                        <h3 class="submission__answer__display__title">{% trans "Your Response" %}</h3>
 
-                        {% trans "Your response to the question above:" as translated_label %}
+                        {% trans "Your Response" as translated_label %}
```

Comment:
Let's be consistent though about whether or not a colon is used. @catong can you weigh in?